README.md

```diff
@@ -32,18 +32,9 @@ http://jenkins.local:8080
 ```
 
 ## First time accessing Jenkins
-Since version 2.0 Jenkins has a security setup wizard when first running it after the installation.
 
-SSH into the machine with
+Jenkins is set up with one user with
+username: admin
```

Comment:
Can you please just put the username and password in a code block, so that it looks a bit nicer :)

Reply:
sure :)